### PR TITLE
Feat/sort and search by ordinal number

### PR DESCRIPTION
**Pull request by habibitcoin**
https://github.com/dannydeezy/nosft/issues/50

### CODE REVIEW

File: src/containers/OrdinalsArea.js [hunk 1]
```diff
@@ -206,6 +209,43 @@ const OrdinalsArea = ({ className, space }) => {
                             )}
                         </button>
                     </div>
+                    <div className="col-lg-1 col-md-1 col-sm-1 col-2">
```
Comment (topether21) [inline src/containers/OrdinalsArea.js:212]: We can create a function in order to keep clean the onClick.
like
`onClick={filterUtxos}`

Comment (andredevjs) [reply src/containers/OrdinalsArea.js:212]: +1

**Comment by andredevjs** [comment]
@habibitcoin this one is good to go.
Can you fix the conflict, so that we can merge it?

**Review by andredevjs — APPROVED** [review]

**Comment by habibitcoin** [comment]
@andredevjs I need to fix 2 things:

1. Sort by value is not working (it appears to try to sort the values but DOM doesnt update)
2. InscriptionNumber is null

File: src/containers/OrdinalsArea.js [hunk 2]
```diff
@@ -244,6 +249,23 @@ const OrdinalsArea = ({ className, space }) => {
...
+                        <button
+                            type="button"
+                            className={clsx(
+                                "sort-button d-flex flex-row justify-content-center",
+                                activeSort === "inscriptionNumber" && "active"
+                            )}
+                            onClick={() => {
+                                sortBy("inscriptionNumber");
+                            }}
```
Comment (StefanBraunovic) [inline src/containers/OrdinalsArea.js:261]: This can cause memory leak . You should extract this into separate  function.

Comment (habibitcoin) [reply src/containers/OrdinalsArea.js:261]: @StefanBraunovic I definitely am experiencing the memory leak now that you mention it, but struggling to patch it. Mind explaining a bit more? I already have "sortBy" as a separate function

Apologies if the question is remedial

**Comment by topether21** [comment]
LGTM! Good job

**Review by topether21 — APPROVED** [review]

**Pull request merged**